pkg/webui/components/tooltip/index.js

```diff
@@ -0,0 +1,83 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import Tippy from '@tippyjs/react'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+
+import './tooltip.styl'
+
+const popperModifiers = [
+  {
+    name: 'offset',
+    options: {
+      offset: [-12, 8],
+    },
+  },
+  {
+    name: 'arrow',
+    options: {
+      element: '.tippy-arrow',
+      padding: 12,
+    },
+  },
+]
+
+const Tooltip = props => {
+  const { className, children, content, interactive, placement } = props
+  return (
+    <Tippy
+      className={className}
+      content={content}
+      interactive={interactive}
+      placement={placement}
+      popperOptions={{ modifiers: popperModifiers }}
+    >
```

Comment:
```suggestion
      delay={300}
    >
```

Let's also use the default fade animation.